src/test/java/org/fcrepo/camel/integration/FcrepoDeleteIT.java

```diff
@@ -182,15 +185,15 @@ public void testDeleteNestedContainer() throws InterruptedException {
         deletedEndpoint.assertIsSatisfied();
 
         // Check deleted container
-        for (Exchange exchange : goneEndpoint.getExchanges()) {
-            Assert.assertTrue(exchange.getIn().getHeader(Exchange.CONTENT_TYPE, String.class).contains("text/html"));
-            Assert.assertTrue(exchange.getIn().getBody(String.class).contains("Gone"));
-        }
-
-        for (Exchange exchange : notFoundEndpoint.getExchanges()) {
-            Assert.assertTrue(exchange.getIn().getHeader(Exchange.CONTENT_TYPE, String.class).contains("text/html"));
-            Assert.assertTrue(exchange.getIn().getBody(String.class).contains("Not Found"));
-        }
+        goneEndpoint.getExchanges().forEach(exchange -> {
+            assertTrue(exchange.getIn().getHeader(Exchange.CONTENT_TYPE, String.class).contains("application/rdf+xml"));
+            assertTrue(exchange.getIn().getBody(String.class).contains("Discovered tombstone"));
+        });
+
+        notFoundEndpoint.getExchanges().forEach(exchange -> {
+            assertTrue(exchange.getIn().getHeader(Exchange.CONTENT_TYPE, String.class).contains("text/html"));
+            assertTrue(exchange.getIn().getBody(String.class).contains("Not Found"));
```

Comment:
Okay by me, although I don't quite why we'd wait for the next release... but it shouldn't make any diff now and we don't want to slow things down even more.